src/cmd/api/controllers/video_upload_handler.go

```diff
@@ -45,13 +46,15 @@ type Response struct {
 // @Accept multipart/form-data
 // @Produce plain
 // @Param file formData file true "video"
+// @Param file formData file false "cover"
+// @Param file formData file true "title"
```

Comment:
Should be : 
// @Param video formData file true "video"
// @Param cover formData file false "cover"
// @Param title formData string true "title"